client/examples/slack-bot/src/index.ts

```diff
@@ -0,0 +1,126 @@
+import { PublicApiClient } from "orbs-interfaces";
+import { initPublicApiClient } from "./public-api-client";
+import { OrbsClientSession, OrbsHardCodedContractAdapter } from "./orbs-client";
+import { FooBarAccount } from "./foobar-account";
+import { RTMClient } from "@slack/client";
+
+interface Config {
+  subscriptionKey: string;
+  publicApiClient: PublicApiClient;
+  timeout: number;
+}
+
+const {
+  SLACK_VERIFICATION_TOKEN,
+  SLACK_TOKEN,
+  CONVERSATION_ID,
+  ORBS_API_ENDPOINT,
+  TRANSACTION_TIMEOUT
+} = process.env;
+
+const BOT_ADDRESS = "0000";
+const PULL_REQUEST_AWARD = 100;
+
+const config = {
+  subscriptionKey: "0x0213e3852b8afeb08929a0f448f2f693b0fc3ebe",
```

Comment:
I don't like that this magic constant appears hard coded in so many places, but that's not the fault of this PR. Let's think how to fix it globally elsewhere.